src/types/IUsers.tsx

```diff
@@ -0,0 +1,10 @@
+export interface IUsers {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {};
+  phone: string;
+  website: string;
+  company: {};
```

Comment:
Either add types for address and company or don't use them at all

Reply:
fixed